src/config/routes.js

```diff
@@ -54,6 +55,10 @@ const Routes = () => {
           <Header props={ pageState.ogsh } />
           <Geneology />
         </Route>
+        <Route path="/demo">
+          <Header props={ pageState.demo } />
+          <Demo />
+        </Route>
```

Comment:
Do you still need this?

Reply:
I figured we can continue using this page for demo purposes. I mainly created it so that I can take the Buttons out of the Donate page.